addon/controllers/new.js

```diff
@@ -6,7 +6,7 @@ export default Ember.Controller.extend({
     post (title, body) {
       let blog = this.get('blog')
       blog.add(title, body)
-      this.transitionToRoute('/blog')
+      this.transitionToRoute('index')
```

Comment:
ah I see. and there is not route named blog? just wondering how it affects the final behaviour if at all